client/container.go

```diff
@@ -52,8 +63,53 @@ func (x *PrmContainerPut) AttachSignature(sig neofscrypto.Signature) {
 	x.sig, x.sigSet = sig, true
 }
 
+var errInvalidNodeSetDesc = errors.New("invalid node set descriptor")
+
+func verifyStoragePolicy(policy netmap.PlacementPolicy) error {
```

Comment:
Can this be a public method of `netmap.PlacementPolicy` for node to reuse?

Reply:
dont mind, will do

Reply:
done